docs/ad/api.md

```diff
@@ -721,14 +773,159 @@ POST _opendistro/_anomaly_detection/detectors/results/_search
     ]
   }
 }
-...
 ```
 
+In historical detectors, specify the `detector_id`:
```

Comment:
How about we change this example to get latest AD result?  
1.Get latest task with get detector API, 
2.then query anomaly results with task id.
```
GET /_opendistro/_anomaly_detection/detectors/results/_search
{
  "query": {
    "term": {
      "task_id": {
        "value": "NnlV9HUBQxqfQ7vBJNzy"
      }
    }
  }
}
```